src/components/StepperDialogWrapper/StepperDialogWrapper.jsx

```diff
@@ -3,17 +3,16 @@ import React from 'react'
 import { useStepperDialogContext } from '../Hooks'
 
 const StepperDialogWrapper = () => {
-  const { allCurrentPages, currentPage } = useStepperDialogContext()
+  const { allCurrentPages, currentPageIndex } = useStepperDialogContext()
 
-  return allCurrentPages.map((page, i) => (
-    <React.Fragment key={i}>
-      {page.pageNumber === currentPage && (
-        <>
-          {/* <StepperDialog ... title={via StepperDialogContext} content={<LazyLoad currentPage={page} />} ... */}
-        </>
-      )}
-    </React.Fragment>
-  ))
+  return allCurrentPages.map(
+    page =>
+      page.pageIndex === currentPageIndex && (
+        <React.Fragment key={page.pageIndex}>
+          {/* <StepperDialog ... title={via StepperDialogContext} content={<LazyLoad currentPageIndex={page} />} ... */}
```

Comment:
Maybe commented out code would be better if deleted. We can still get it back from git history

Reply:
Effectivement, je l'avais laissé pour "améliorer" la compréhension lors de la review, je vais le supprimer 😁